Tests/Marketplace/upload_packs.py

```diff
@@ -675,8 +676,14 @@ def main():
             pack.cleanup()
             continue
 
-        # in case that pack already exist at cloud storage path, skipped further steps
-        if skipped_pack_uploading:
+        task_status, exists_in_index = pack.check_if_exists_in_index(index_folder_path)
+        if not task_status:
+            pack.status = PackStatus.FAILED_SEARCHING_PACK_IN_INDEX.name
```

Comment:
why is that not inside the pack method?

Reply:
What do you mean? It is pack method

Reply:
why not update the status inside the `check_if_exists_in_index` function?